docs/folder-structure.md

```diff
@@ -0,0 +1,96 @@
+# F36 Folder Structure
+
+To make it easy to maintain our files and folders we ask every contributor to follow this folder structure.
+Every package has its own folder, except for components which should be kept in the folder `/components` (every component must have its own folder).
```

Comment:
I am not sure if I understand this sentence. 🤔 
Maybe we could split it somehow, to clearly distinguish the concepts - packages vs components? For example:
"Every package has its own folder. _Here we could give some examples maybe?_" 
"New components should be kept in the folder `/components` and has its own folder as well."

Reply:
What I meant was that:

package = every folder with a package.json file basically... and that we publish in npm as a package

Inside `/packages` we have: core, f36-website, f36-tokens, etc

So every package we create we create a folder for it in `/packages`, except when it's a react ui component
then we create a folder inside of `/packages/components`. (e.g. button, accordion, autocomplete, forms)

Reply:
How can I explain that ☝️ better?

Reply:
Maybe just be more verbose? 🤔 
Something like: 
Every package that should be published should have it's own folder, with the exception of UI components, that should have their folder and files created inside the `components` directory.

Reply:
yup! Sounds good!